pkg/meta/base.go

```diff
@@ -121,6 +121,8 @@ type engine interface {
 	doSetFacl(ctx Context, ino Ino, aclType uint8, rule *aclAPI.Rule) syscall.Errno
 	doGetFacl(ctx Context, ino Ino, aclType uint8, aclId uint32, rule *aclAPI.Rule) syscall.Errno
 	cacheACLs(ctx Context) error
+
```

Comment:
base on comment 1, remove this?
we need one new API only

Reply:
still a bit confused. I believe that this interface is essential. Based on the current code structure, we can't handle all kvmeta operations within baseMeta. Could you provide some specific suggestions or guidance on how to approach this? I'm having trouble figuring it out.

Reply:
My apologies for not being clear enough.
- move engine API NewMetaEntryScanner
- keep Meta API NewBaseEntryScanner (maybe change the name to NewEntryScanner , would be clearly enough)
- move the implement in baseMeta to kvMeta, merge them

Reply:
thanks for your hints, now i remove this unnecessary **BaseEntryScanner**.